gp-templates/project-import.php

```diff
@@ -24,9 +28,28 @@
 		$format_options[$slug] = $format->name;
 	}
 	$format_dropdown = gp_select( 'format', $format_options, 'auto' );
+
+	$status_options = array();
+	if ( isset( $can_import_current ) && $can_import_current ) {
+		$status_options['current'] = __( 'Current', 'glotpress' );
+	}
+	if ( isset( $can_import_waiting ) && $can_import_waiting ) {
+		$status_options['waiting'] = __( 'Waiting', 'glotpress' );
+	}
 ?>
-	<dt><label	for="format"><?php _e( 'Format:', 'glotpress' ); ?></label></dt>
+	<dt><label for="format"><?php _e( 'Format:', 'glotpress' ); ?></label></dt>
 	<dd><?php echo $format_dropdown; ?></dd>
+<?php if ( ! empty( $status_options ) ) : ?>
+	<dt><label for="status"><?php _e( 'Status:', 'glotpress' ); ?></label></dt>
+	<dd>
+		<?php if ( count( $status_options ) === 1 ) : ?>
+			<input type="hidden" name="status" value="<?php echo esc_attr( reset( array_keys( $status_options ) ) ); ?>" />
+			<?php echo esc_html( reset( array_values( $status_options ) ) ); ?>
+		<?php elseif ( count( $status_options ) > 1 ) : ?>
+			<?php echo gp_select( 'status', $status_options, 'waiting' ); ?>
```

Comment:
Should the default value be 'current' instead of 'waiting' to match the existing behaviour?